distrib/wix/README

```diff
@@ -10,8 +10,10 @@ was used for testing.
 Assuming WiX tools are in the path, simply run the following commands to
 produce bkl.msi:
 
+	heat dir ..\..\docs\html\ -srd -ag -sfrag -template fragment -dr DOCSDIR -cg docs -var var.DocSource -out docs.wxs
```

Comment:
I'd rather avoid duplicating makefile commands here, this was acceptable as long as it was a single simple one, but now that it's not the case any more, let's just refer the reader to the makefile instead and just put `make distrib_msi` here.